pages/signin.tsx

```diff
@@ -38,37 +67,79 @@ const SignIn: NextPage = () => {
 								handleSignIn();
 							}}
 						>
-							<div className='flex w-[90%] flex-col space-y-2'>
-								<p>Email</p>
+							<div className='flex w-full flex-col space-y-3'>
+								<label htmlFor='email-address' className='text-sm text-gray-600'>
+									Email address
+								</label>
 								<input
 									type='text'
+									id='email-address'
 									placeholder='example@gmail.com'
 									required
-									className='input-field outline-none'
+									className='relative block w-full appearance-none rounded-md border border-gray-300 px-3 py-2 text-gray-900 placeholder-gray-500 focus:z-10 focus:border-indigo-500 focus:outline-none focus:ring-indigo-500 sm:text-sm'
 									value={email}
 									onChange={e => setEmail(e.target.value)}
 								/>
 							</div>
-							<div className='flex w-[90%] flex-col space-y-2 '>
-								<p>Password</p>
+							<div className='flex w-full flex-col space-y-2 '>
+								<label htmlFor='password' className='text-sm text-gray-600'>
+									password
+								</label>
 								<input
 									type='password'
+									id='password'
 									required
-									className='input-field outline-none'
+									className='relative flex w-full appearance-none justify-center rounded-md border border-gray-300 px-3 py-2 text-gray-900 placeholder-gray-500 focus:z-10 focus:border-indigo-500 focus:outline-none focus:ring-indigo-500 sm:text-sm'
 									placeholder='*****'
 									value={password}
 									onChange={e => setPassword(e.target.value)}
 								/>
 							</div>
+							<div className='flex items-center justify-between'>
+								<div className='flex items-center'>
+									<input
+										id='remember-me'
+										name='remember-me'
+										type='checkbox'
+										className='h-4 w-4 rounded border-gray-300 text-indigo-600 focus:ring-indigo-500'
+									/>
+									<label htmlFor='remember-me' className='ml-2 block text-sm text-gray-900'>
+										{' '}
+										Remember me{' '}
```

Comment:
do we have these functionalities yet?

Reply:
(if not, let's comment them out or delete them)

Reply:
I don't think that functionality has been implemented yet. i'll comment out that part